qiling/os/windows/const.py

```diff
@@ -12,12 +12,16 @@
 ERROR_PATH_NOT_FOUND = 0x3
 ERROR_ACCESS_DENIED = 0x5
 ERROR_INVALID_HANDLE = 0x6
+ERROR_GEN_FAILURE = 0x1f
```

Comment:
I think here should be capitalized?

Reply:
I don't think we have a consistent convention for that (there are many lowercase and upercase constans everywhere), neither PEP8 defines it. Is there are specific reason here?

Reply:
No, I just saw the variables that existed before is all uppercase values.